newrelic/resource_newrelic_alerts_location_failure_condition.go

```diff
@@ -0,0 +1,208 @@
+package newrelic
+
+import (
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/newrelic/newrelic-client-go/pkg/alerts"
+	"github.com/newrelic/newrelic-client-go/pkg/errors"
+)
+
+func resourceNewRelicAlertsLocationFailureCondition() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceNewRelicAlertsLocationFailureConditionCreate,
+		Read:   resourceNewRelicAlertsLocationFailureConditionRead,
+		Update: resourceNewRelicAlertsLocationFailureConditionUpdate,
+		Delete: resourceNewRelicAlertsLocationFailureConditionDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"policy_id": {
+				Type:        schema.TypeInt,
+				Required:    true,
+				ForceNew:    true,
+				Description: "The ID of the policy where this condition should be used.",
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The title of this condition.",
+			},
+			"runbook_url": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Runbook URL to display in notifications.",
+			},
+			"enabled": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+				Description: "Set whether to enable the alert condition. Defaults to true.",
+			},
+			"entities": {
+				Type:        schema.TypeList,
+				Required:    true,
+				MinItems:    1,
+				Description: "The GUIDs of the Synthetics monitors to alert on. Defaults to true.",
```

Comment:
Defaults to True?

Reply:
The tests will have me fix all this soon enough.